scripts/core/editor3/helpers/highlights.js

```diff
@@ -363,7 +363,18 @@ export function getHighlightDataAtCurrentPosition(editorState, types) {
  * @description add a new highlight for the current selection.
  */
 export function addHighlight(editorState, type, data) {
-    const highlightsState = getHighlightsState(editorState);
+    var nextEditorState = editorState;
```

Comment:
You're assigning nextEditorState to editorState here but reassigning on the next line, should it be like this?:

```diff
- var nextEditorState = editorState;
+ const nextEditorState = EditorState.acceptSelection(
+   editorState,
+   editorState.getSelection().merge({
+     hasFocus: true
+   })
+ );
```

Reply:
You are right, there's no need to reassign here :)